### PR TITLE
Pollo Tropical

### PR DESCRIPTION
**Pull request by vanshv**
Added spider for Pollo Tropical #3513 
[https://www.pollotropical.com/locations](url)

### CODE REVIEW

**Comment by mjoe999** [comment]
Pollo Tropical item_attributes could be improved and include wikidata code, from helper command:

$ pipenv run scrapy nsi --name pollotropical
"Pollo Tropical", "Q3395356"
       -> https://www.wikidata.org/wiki/Q3395356
       -> https://www.wikidata.org/wiki/Special:EntityData/Q3395356.json
       -> fast food restaurant chain
       -> https://www.pollotropical.com/
       -> **item_attributes = {"brand": "Pollo Tropical", "brand_wikidata": "Q3395356"}**
       -> {'displayName': 'Pollo Tropical', 'id': 'pollotropical-658eea', 'locationSet': {'include': ['001']}, 'tags': {'amenity': 'fast_food', 'brand': 'Pollo Tropical', 'brand:wikidata': 'Q3395356', 'cuisine': 'chicken', 'name': 'Pollo Tropical', 'takeaway': 'yes'}}

**Comment by vanshv** [comment]
do share if any other improvements could be made, this is my first time makind a spider.

File: locations/spiders/pollotropical.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+class QuickleesSpider(scrapy.Spider):
```
Comment (mjoe999) [inline locations/spiders/pollotropical.py:6]: Rename to PolloTropicalSpider

File: locations/spiders/pollotropical.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    name = "pollotropical"
+    allowed_domains = ["https://www.pollotropical.com/"]
+    item_attributes = {
+        "brand": "Pollo Tropical",
+        "brand_wikidata": "Q3395356",
+        "country": "USA",
```
Comment (mjoe999) [inline locations/spiders/pollotropical.py:12]: This adds country to yielded POI. Please use ISO-2 code: "US" not "USA"
Also you duplicate this by setting country further down. One or the other but not both!

File: locations/spiders/pollotropical.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+
+class QuickleesSpider(scrapy.Spider):
+    name = "pollotropical"
+    allowed_domains = ["https://www.pollotropical.com/"]
```
Comment (mjoe999) [inline locations/spiders/pollotropical.py:8]: You're only asking scrapy to go to one URL which you ave specfified. Probably overkill to set allowed_domains!

File: locations/spiders/pollotropical.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    start_urls = (
+        'https://www.pollotropical.com/locations',
+    )
+
+    def parse(self, response):
+
+        store = response.xpath(
+            "//div[@class='styles__StyledCardInfo-s1y7dfjk-3 fncRsG']/div/h3/a/text()").extract()
```
Comment (mjoe999) [inline locations/spiders/pollotropical.py:22]: This look really brittle. If I look at the web page you attempt to scrape there is a big JSON array:
**<script>__NEXT_DATA__ = {"props":{"pageProps":{"isServer":true,"store .......**
that you could load and walk through. The array has lat/lon points also.
Use dict_parser.DictParser() ... look for examples of use in the project ... to help you parse each store JSON?

**Pull request merged**